resources/views/atom/user.blade.php

```diff
@@ -43,7 +43,6 @@
                     <media:content
                         url="{{ $media['url'] }}"
                         type="{{ $media['mime'] }}"
-                        medium="image"
```

Comment:
Since not only images can be found here, let the feed reader decide how to display media.